### ISSUE TITLE
Running a 64 bit version of JPCSP in 32 bit JRE in 64 bit Machine

### ISSUE THREAD

**Issue by SonicGDI** [jpcsp/jpcsp#534]
I installed the whole 32 bit and 64 bit jre, and i can't run the emulator, is there anyway to fix it?

**Comment by hlide** [comment]
I don't think you can mix 64-bit and 32-bit this way. Running a 64-bit version of JPCSP is asking a 64-bit version of JRE, especially for a 64-bit machine. Install the 64-bit version if JRE. 


De: "SonicGDI" ***@***.***> 
À: "jpcsp/jpcsp" ***@***.***> 
Cc: "Subscribed" ***@***.***> 
Envoyé: Mardi 24 Septembre 2024 05:29:05 
Objet: [jpcsp/jpcsp] Running a 64 bit version of JPCSP in 32 bit JRE in 64 bit Machine (Issue #534) 





I installed the whole 32 bit and 64 bit jre, and i can't run the emulator, is there anyway to fix it?
